app_dart/lib/src/request_handlers/luci_status.dart

```diff
@@ -73,7 +73,7 @@ class LuciStatusHandler extends RequestHandler<Body> {
       json.decode(requestString) as Map<String, dynamic>,
     );
     final BuildPushMessage buildPushMessage =
-        BuildPushMessage.fromJson(json.decode(envelope.message!.data!) as Map<String, dynamic>);
+        BuildPushMessage.fromJson(json.decode(String.fromCharCodes(base64.decode(envelope.message!.data!))) as Map<String, dynamic>);
```

Comment:
I'm not sure why this is now surfacing as an issue in the new Dart version.

`envelope.message.data` has always been a base64 encoded string of the json, however it seems json decode removed the base64 decode aspect